aemanalyser-core/src/main/java/com/adobe/aem/analyser/Runmode.java

```diff
@@ -0,0 +1,26 @@
+/*************************************************************************
+ * ADOBE CONFIDENTIAL
```

Comment:
Rather name this `ServiceType`, run mode is IMHO a broader concept and includes more dimensions than just author/publish.

Reply:
I've made this change, thanks for the suggestion.